xml/ReportingOrganisation.xml

```diff
@@ -9162,7 +9162,7 @@
             <codeforiati:organisation-type-code>21</codeforiati:organisation-type-code>
             <codeforiati:organisation-type>International NGO</codeforiati:organisation-type>
             <codeforiati:hq-country-or-region>Sweden</codeforiati:hq-country-or-region>
-            <codeforiati:registry-identifier>diakonia802017-3517</codeforiati:registry-identifier>
+            <codeforiati:registry-identifier>diakonia-sweden</codeforiati:registry-identifier>
```

Comment:
Not really keen on registry identifiers changing. I ought to add these to a mapping file when this happens.